crates/parser/veryl.par

```diff
@@ -778,14 +778,16 @@ WithParameterItem: ( Param | Const ) Identifier Colon ( ArrayType | Type ) Equ E
 GenericBound: Const

```

Comment:
Is there any reason to keep `Const`?

Reply:
It is just compatibility for existing code.

Reply:
I think it is better to introduce breaking change now than later.

Reply:
OK, I understood.
I will rmeove `const` generic boundry in this PR.

Reply:
> I will rmeove const generic boundry in this PR.

I've removed `const` generic boundary.
ae4aa6a1257773eb3f91ea0ced63b71e3f5d4083